src/Packetery/Module/Order/Metabox.php

```diff
@@ -281,7 +281,7 @@ public function save_fields( $orderId ) {
 		}
 
 		$propsToSave = [
-			self::FIELD_WEIGHT => ( is_numeric( $values[ self::FIELD_WEIGHT ] ) ? Helper::simplifyWeight( $values[ self::FIELD_WEIGHT ] ) : null ),
+			self::FIELD_WEIGHT => ( is_numeric( $values[ self::FIELD_WEIGHT ] ) ? (float) number_format( $values[ self::FIELD_WEIGHT ], 0, '.', '' ) : null ),
```

Comment:
Proč nula? Proč je potřeba to zde formátovat? Dle mého stačí zde mít float nebo null. Pokud je nutné to formátovat tak použij simplifyWeight().

Reply:
Nula je omyl, opraveno. Formátování tam bylo pro jednotnost s ostatními hodnotami, kde je možná také zbytečné. simplifyWeight by zde bylo zbytečné, používá se v setteru.